### DISCUSSION TITLE
Daily Standup - Wednesday, April 26

### DISCUSSION THREAD

**Discussion by antonieto** [antonieto/ibm-dashboard#146] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Worked on auth view
 - Worked on architecture document

**What are your priorities for the next 24 hours?**:
 - Keep working on view

**Blockers**:
 - Been really lazy lately

**Shoutouts**:
 - @everyone for finishing quickly our first checkpoint

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Started working on the Architecture Document
 - Researched how can layer architecture be implemented in NextJS

**What are your priorities for the next 24 hours?**:
 - Implement #136 
 - Advance #138 

**Blockers**:
 - None

**Shoutouts**:
 - @DavBot02 for joining

**Comment by Jhoscar22** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished Next.js introduction course + typescript

**What are your priorities for the next 24 hours?**:
 - Implement signup

**Blockers**:
 - None

**Shoutouts**:
 - @antonieto for establishing clear Git practices for the project.

**Comment by Int-Angel** [comment]
**What did you achieve in the last 24 hours?**:
 - Started working doc arquitecture
 - Fixed PR comments DB_test
 - Finished boards screen

**What are your priorities for the next 24 hours?**:
 - Update to styled components

**Blockers**:
 - we need to deploy the db

**Shoutouts**:
 - @antonieto for explaining git practices
